src/music/Notes.ts

```diff
@@ -0,0 +1,36 @@
+import { mod } from "../util/Util.js";
+
+export interface Note {
+
+    note: number; // the MIDI number of the note
```

Comment:
Per the comment above, this is a terrible confusing field name. If it's a MIDI number, call it a `midiNumber`.

Reply:
Also, this should be newtyped at some point.